Assets/HoloToolkit/Utilities/README.md

```diff
@@ -14,6 +14,16 @@ A drop in direction indicator that stays in the users view at all times.
 #### HeadsUpDirectionIndicatorPointer.prefab
 A quad based pointer to be used with the HeadsUpDirectionIndicator prefab to create an out of box direction indicator.
 
+###[Scenes](Scenes)
+---
+
+####HeadsetAdjustment.unity
```

Comment:
Utilities\Tests\Scenes basically. We don't want to *force* app developers to use this and hence I recommend putting this under Tests.